Formula/openjdk.rb

```diff
@@ -60,61 +63,55 @@ def install
       MacOS::Xcode.prefix,
     )
 
-    boot_jdk_dir = Pathname.pwd/"boot-jdk"
-    resource("boot-jdk").stage boot_jdk_dir
-    boot_jdk = boot_jdk_dir/"Contents/Home"
-    java_options = ENV.delete("_JAVA_OPTIONS")
+    # Get build number from URL
+    build = /#{version}\+(\d+)\.tar\.gz/.match(active_spec.url)[1]
 
-    # Inspecting .hgtags to find a build number
-    # The file looks like this:
-    #
-    # 1613004c47e9dc867a2c2c43d716533b1aaedc5f jdk-15.0.2+0
-    # cc4fdb537bc14734064a9a8eadb091fd1c12b36e jdk-15.0.2+1
-    # d24e907486b3f90691980b0dde01efca5840abc6 jdk-15.0.2+2
-    # dbb11e11955ad1240ba775ab0007a14547e14ce6 jdk-15.0.2+3
-    # 4c4a2eb7b19ecb31620e6bb120e40f8a5fd1737a jdk-15.0.2+4
-    # e431a9461b1356c4b763443e5333b3f4a8695eaf jdk-15.0.2+5
-    # d5977ee56509ceaa3d3c8e1aebbca76651358da4 jdk-15.0.2+6
-    # 38912b2a5bcb396c75f8707e300773c874327451 jdk-15.0.2+7
-    #
-    # Since openjdk has move their development from mercurial to git and GitHub
-    # this approach may need some changes in the future
-    #
-    version_to_parse = if Hardware::CPU.arm?
-      "16"
-    else
-      version
+    boot_jdk = Pathname.pwd/"boot-jdk"
+    resource("boot-jdk").stage boot_jdk
+    on_macos do
+      boot_jdk /= "Contents/Home"
     end
-    build = File.read(".hgtags")
-                .scan(/ jdk-#{version_to_parse}\+(.+)$/)
-                .map(&:first)
-                .map(&:to_i)
-                .max
-    raise "cannot find build number in .hgtags" if build.nil?
+    java_options = ENV.delete("_JAVA_OPTIONS")
 
     args = %W[
-      --without-version-pre
-      --without-version-opt
-      --with-version-build=#{build}
-      --with-toolchain-path=/usr/bin
-      --with-boot-jdk=#{boot_jdk}
+      --disable-warnings-as-errors
       --with-boot-jdk-jvmargs=#{java_options}
+      --with-boot-jdk=#{boot_jdk}
       --with-debug-level=release
-      --with-native-debug-symbols=none
       --with-jvm-variants=server
-      --with-sysroot=#{MacOS.sdk_path}
-      --with-extra-ldflags=-headerpad_max_install_names
-      --enable-dtrace
+      --with-native-debug-symbols=none
+      --with-vendor-bug-url=#{CoreTap.instance.issues_url}
```

Comment:
Why not this, it's nicer when people extract it.
```suggestion
      --with-vendor-bug-url=#{tap.issues_url}
```